hoot-core/src/main/cpp/hoot/core/elements/Way.cpp

```diff
@@ -226,6 +226,15 @@ bool Way::isOneWay() const
   return result;
 }
 
+bool Way::isSimpleLoop() const
+{
+  if (getNodeId(0) == getNodeId(getNodeCount()-1))
```

Comment:
This looks good.  I'm a simple kind of man:
```
  return (getNodeId(0) == getNodeId(getNodeCount()-1);
```
But like Seth says "it'll probably be optimized out" so it is up to you.